commands/try-runtime/try-runtime.sh

```diff
@@ -20,27 +26,25 @@ main() {
       die "the network should be provided"
   fi
 
-  local preset_args=(
-    run
-    # Requirement: always run the command in release mode.
-    # See https://github.com/paritytech/command-bot/issues/26#issue-1049555966
-    --release
-    # "--quiet" should be kept so that the output doesn't get polluted
-    # with a bunch of compilation stuff
-    --quiet
-    --features=try-runtime
-    try-runtime
-    --chain="${network}-dev"
-    --execution=Wasm
-    --no-spec-check-panic
-    on-runtime-upgrade
-    live
-    --uri wss://${network}-try-runtime-node.parity-chains.parity.io:443
-  )
-
   set -x
   export RUST_LOG="${RUST_LOG:-remote-ext=debug,runtime=trace}"
-  cargo "${preset_args[@]}" "$@"
+
+  # following docs https://paritytech.github.io/substrate/master/try_runtime_cli/index.html
+  cargo build --release
+  cp "./target/release/${repository}" .
```

Comment:
```suggestion
```

Reply:
is being done in line 38 as well?

Reply:
ah, this was from docs, but indeed, not sure why this files was copied 
<img width="963" alt="image" src="https://user-images.githubusercontent.com/1177472/210171621-b12fd88a-b758-44b9-b8bb-fda89f4b800c.png">